app/views/doc/credential_helpers.html.erb

```diff
@@ -0,0 +1,48 @@
+<%- @section    = 'documentation' %>
+<%- @subsection = 'credential-helpers' %>
+<%- @page_title = 'Git credential helpers' %>
+
+<div id="main">
+  <h1>Git credential helpers</h1>
+
+  <p>This page lists available <a href="https://git-scm.com/docs/gitcredentials">credential helpers</a>.</p>
+
+  <h2>Included in Git</h2>
+
+  <ul>
+    <li><a href="https://git-scm.com/docs/git-credential-cache">git-credential-store</a>: saves credentials in plaintext.</li>
+    <li><a href="https://git-scm.com/docs/git-credential-cache">git-credential-cache</a>: holds credentials temporarily in process memory. <a href="https://github.com/git-for-windows/git/issues/3892">Not available on Windows</a>. (NB. Since credentials are lost when the cache expires, this is inconvenient to store personal access tokens.)</li>
+  </ul>
```

Comment:
I wonder if we should place these further down in the list; most users don't use or need to be familiar with these built-in credential helpers.